numpyro/contrib/tfp/distributions.py

```diff
@@ -234,22 +258,9 @@ def _onehot_enumerate_support(self, expand=True):
     if _Dist is tfd.Distribution:
         continue
 
-    try:
-        _PyroDist = locals()[_name]
-    except KeyError:
-        _PyroDist = TFPDistribution[_Dist]
-        _PyroDist.__module__ = __name__
-        if hasattr(numpyro_dist, _name):
-            numpyro_dist_class = getattr(numpyro_dist, _name)
-            # resolve FooProbs/FooLogits namespaces
-            if type(numpyro_dist_class).__name__ == "function":
-                if not hasattr(numpyro_dist, _name + "Logits"):
-                    continue
-                numpyro_dist_class = getattr(numpyro_dist, _name + "Logits")
-            _PyroDist.arg_constraints = numpyro_dist_class.arg_constraints
-            _PyroDist.has_enumerate_support = numpyro_dist_class.has_enumerate_support
-            _PyroDist.enumerate_support = numpyro_dist_class.enumerate_support
-        locals()[_name] = _PyroDist
+    _PyroDist = TFPDistribution[_Dist]
+    _PyroDist.__module__ = __name__
+    locals()[_name] = _PyroDist
```

Comment:
nit: could you also help me remove the docstring below
```
for name in __all__[:_len_all]   # this is not required: + sorted(__all__[_len_all:])
```

Reply:
As in, keep the docstring, but don't document any of the generated wrapped distributions? Believe I have done this but do check to make sure I understood correctly.